debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/RecordsStreamProducer.java

```diff
@@ -249,30 +251,31 @@ private void process(ReplicationMessage message, Long lsn, BlockingConsumer<Chan
             logger.debug("received new message at position {}\n{}", ReplicationConnection.format(lsn), message);
         }
 
-        TableSchema tableSchema = tableSchemaFor(tableId);
-        if (tableSchema != null) {
```

Comment:
So everything above this is what I am unsure of. I cannot get the table because there are no changes with table information etc. I think these changes are safe but I may be missing something. There is the "assert tableId != null;" but the SourceInfo using this specifically says "@param tableId the table that should be included in the source info; may be null". Then I don't understand the "tableSchema != null" because nothing in this method is actually using tableSchema.